### PR TITLE
Add compatibility for TYPO3 9

### PR DESCRIPTION
**Pull request by flossels**

### CODE REVIEW

File: Classes/OpenidService.php [hunk 1]
```diff
@@ -518,7 +521,7 @@ protected function normalizeOpenID($openIDIdentifier)
      */
     protected function getBackPath()
     {
-        $extPath = ExtensionManagementUtility::siteRelPath('openid');
+        $extPath = ExtensionManagementUtility::extPath('openid');
```
Comment (liayn) [inline Classes/OpenidService.php:524]: Did you test this?
extPath returns an absolute path, so that will most probably not work out

**Comment by flossels** [comment]
yeah, I have tested that.
`ExtensionManagementUtility::siteRelPath('openid')` will return `typo3conf/ext/openid/lib/php-openid` wich is the same result as `substr(PathUtility::getAbsoluteWebPath(ExtensionManagementUtility::extPath('openid') . 'lib/php-openid'),1)`; both does not work.

File: Classes/OpenidService.php (hunk 1 above)
Comment (liayn) [reply Classes/OpenidService.php:524]: > `ExtensionManagementUtility::siteRelPath('openid')` will return `typo3conf/ext/openid/lib/php-openid`

ehm what? what's going on here?

Comment (dmitryd) [reply Classes/OpenidService.php:524]: Append after line 524 to get the same as from `siteRelPath`:
```php
$extPath = substr($extPath, strlen(Environment::getPublicPath() . '/'));
```

**Comment by dmitryd** [comment]
Currently I need this ext for our 9.5.0 project, so I am updating it. I will make it compatible with both 8.7.x and 9.5.0 and will make a PR.

**Comment by liayn** [comment]
Integrated in #23

**Pull request closed without merging**